packages/contracts/migrations/mainnet/Deploy002_Governance.s.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {Script} from "forge-std/Script.sol";
+import {UbiquityAlgorithmicDollarManager} from "../../src/old/UbiquityAlgorithmicDollarManager.sol";
+import {UbiquityGovernance} from "../../src/old/UbiquityGovernance.sol";
+
+/// @notice Migration contract
+/// @dev This migration is used only for reference that we use already deployed
+/// UbiquityAlgorithmicDollarManager and UbiquityGovernance contracts
+contract Deploy002_Governance is Script {
+    // contracts
+    UbiquityAlgorithmicDollarManager ubiquityAlgorithmicDollarManager;
+    UbiquityGovernance ubiquityGovernance;
+
+    function run() public virtual {
+        // use already deployed UbiquityAlgorithmicDollarManager
+        ubiquityAlgorithmicDollarManager = UbiquityAlgorithmicDollarManager(
+            0x4DA97a8b831C345dBe6d16FF7432DF2b7b776d98
+        );
+
+        // use already deployed Governance token
```

Comment:
on which network? should i make the idea is mainnet?, but let's say i am testnet dev, no contracts there (yet)

Reply:
Updated https://github.com/ubiquity/ubiquity-dollar/pull/842/commits/b4791ffb0e50559b1c0cbf10bc71b1bbf566c19b